eduNEXT/xblock-limesurvey
# feat: add limesurvey iframe

### Description
This PR adds a student's survey as an iframe

### How to test
1. [Enable](https://manual.limesurvey.org/Global_settings#Security) the IFrame in the global settings of LimeSurvey (`Configuration > Global > Security > IFrame embedding allowed`) Set it to 'Allow'. 
2. Configure a survey as in this [PR](https://github.com/eduNEXT/xblock-limesurvey/pull/3).
3. From the LMS access to the unit. You should see the embedded survey.
![image](https://github.com/eduNEXT/xblock-limesurvey/assets/64033729/67496576-78a8-4482-99a7-085da840bca8)
4. From the CMS, you should see a simple message.
![image](https://github.com/eduNEXT/xblock-limesurvey/assets/64033729/f54851fa-a9d7-418e-9c69-7c6532e1da7f)
